docs/source/AdministratorGuide/ServerInstallations/InstallingWebAppDIRAC.rst

```diff
@@ -158,35 +158,40 @@ The structure of the web.cfg file is the following::
       {
```

Comment:
Apart that this reminds me of https://github.com/DIRACGrid/WebAppDIRAC/issues/368, I thought that this specific part was not needed anymore...?

Reply:
**v4r2** is still trying to find and make `gConfig.loadCFG` from `etc/web.cfg` or `WebApp/web.cfg`. Probably, it is necessary to make separate PR where to remove this association with `web.cfg` and to test behavior of a portal.